### PR TITLE
fix: remove show case in homepage

### PR DESCRIPTION
**Pull request by liuxiran**
Changes:

Remove showcase in homepage, and we should redesign this part

### CODE REVIEW

File: website/src/pages/index.js [hunk 1]
```diff
@@ -113,59 +113,6 @@ const LearnHow = () => (
   </div>
 );
 
-const Showcase = () => {
-  const { siteConfig } = useDocusaurusContext();
-  if (!(siteConfig.customFields.showcases || []).length) {
-    return null;
-  }
-  const showcases = siteConfig.customFields.showcases.map((user) => (
```
Comment (juzhiyuan) [inline website/src/pages/index.js:121]: `siteConfig.customFields.showcases` Do we need this?

**Comment by 1502shivam-singh** [comment]
Hey @liuxiran , checkout #519 I shared some perspective from my side on this. Good step to take I suppose 👍

**Comment by liuxiran** [comment]
this pr will be ready when our showcase page online

File: website/src/pages/index.js (hunk 1 above)
Comment (liuxiran) [reply website/src/pages/index.js:121]: yes, we will reuse it in our new show case page

**Comment by liuxiran** [comment]
this can be merged after https://github.com/apache/apisix-website/pull/547 merged

**Comment by liuxiran** [comment]
since #547 has been merged, this pr marked ready now, please have a look when you have time, thanks

also cc @1502shivam-singh @Baoyuantop @guoqqqi @iamayushdas to review

**Review by iamayushdas — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**